Development.md

```diff
@@ -0,0 +1,248 @@
+- Publisher, a javascript runtime that reads the xai-node container logs and posts assertions data to the CDN
+
+The public node can be setup on any host with enough resources to run two arbitrum nodes. The host setup can be done through terraform. The configs needed are in `infrastructure\nitro-node-wrapper`, the publisher javascript runtime at `infrastructure\nitro-node-wrapper\publisher\index.mjs` 
+
+Specific environment variables have to be provided in a `.env`:
+```
+# Arb1 Node Settings
+# The Etehreum mainnet RPC to sync the ARB1 node with (prob > 100K requests to sync)
+ETH_RPC_URL= 
+
+# GBUCKET Project ID
+PROJECT_ID=
+# GBUCKET Account Email
+SERVICE_ACCOUNT_EMAIL=
+# GBUCKET PRIVATE KEY
+SERVICE_ACCOUNT_PRIVATE_KEY=
+# GBUCKET BUCKET NAME
+BUCKET_NAME=xai-sentry-public-node
+```
+
+Using the XAI Quicknode ETH RPC - Quicknode access required
+Google Cloud Storage `xai-sentry-public-node` access required
+
+Assertions currently get published to the Google Cloud Bucket https://console.cloud.google.com/storage/browser/xai-sentry-public-node/assertions
+
+Currently the public node is setup at https://console.cloud.google.com/compute/instancesDetail/zones/us-east4-b/instances/arbitrum-full-node-us?hl=de&project=xai-watchtower-node (Google Cloud access required for project `xai-watchtower-node`)
+
+**Manage the containers** (needs user to be in docker group or use `sudo`)
+- Stop all containers gracefully, in `/opt/public-node/build` `docker stop --time=300 $(docker ps -aq) && docker compose down`
+- Restart public-node-stack, in `/opt/public-node/build` `docker compose up -d` (requires .env in directory)
+- View logs, in `/opt/public-node/build` `docker compose logs -f -n 100` to view the last 100 logs and follow
+
+The xai node and arb node data will be stored in `/opt/public-node/data`
+
+
+# Deployed Runtimes
+
+## Challenger
+
+The sentry CLI running `boot-challenger` on a google cloud compute VM instance 
+https://console.cloud.google.com/compute/instances?hl=de&project=xai-sentry-node-challenger
+
+Google Cloud access to project `xai-sentry-node-challenger` required
+
+Running in a `screen` session running the CLI on the VM.
+Needs the "challenger" privateKey (`0xc74c1e08963ceef0e1f2f2a2eeb879f443e86836`) and bls privateKey (`0x977f9fe1d570ba7e48ea76566718023b260ed56eba368ff20dca06e99e650804126663c1a5febdb065ef139a0156a9ad`) on startup
+
+## Data sync
+
+The sentry CLI running `start-centralization-runtime` on a google cloud compute VM instance
+https://console.cloud.google.com/compute/instancesDetail/zones/us-central1-c/instances/backup-runtime?hl=de&project=xai-watchtower-node
+
+This is syncing the mongoDB used for the staking app on events emitted from the PoolFactory as well as updating the staking rewards from subgraph-data on every challenge.
+
+Google Cloud access to project `xai-watchtower-node` required
+
+Running a docker container with a `screen` session running the CLI for each runtime.
```

Comment:
This could be explained a bit better. Possibly:

Each runtime is in it's own docker container. Each container has a 'screen' session running that you may connect to in order to interact with the container.

Also, we should note here the importance of how to "exit" a screen to prevent the container shutting down.

Reply:
Added screen info and added the info line.